packages/orbit-components/src/icons/svg/twitter.svg

```diff
@@ -1,4 +1,4 @@
-<!--character:1007-->
+<!--character:0007-->
```

Comment:
Curious about this change, do you know what caused this?

Reply:
I manually changed this 🙈 we can't have repeated `character` ids, but they are not really "scientific". And since we were gonna remove it soon, I just changed this to an unexisting number

Reply:
Just for me to understand the relevance: how/where is this information used? :D